### PR TITLE
Project

### PR DESCRIPTION
**Pull request by emyrue**
Link to video: https://www.loom.com/share/90eedc1978cb45fab8504ecd6b2b1c5e
Made a website for an imaginary company that hosts concerts.

- Made a popup menu for the mobile version
- Made a fixed menu for the desktop version
- Made links to the about page in both versions
- Headline section, information section, featured artists section, partners section, etc. 
- Made links for past concerts on the about page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (MahmoudBakr23) [inline README.md:52]: - [ ] Please fix the broken MIT link for a completely professional README file and repo 😄

File: README.md [hunk 2]
```diff
@@ -1 +1,52 @@
-# Module1Capstone
\ No newline at end of file
+# Capstone Project
+
+> In this capstone project you are going to build is based on an online website for a conference. We provide some design guidelines for you to create the website, but you must personalize the content, i.e., instead of a conference you can build a website for a concert or for a web development course. It's very important the you personalize your project so that you have something unique in your portfolio to share with potential employers during job searching.
+
```
Comment (MahmoudBakr23) [inline README.md:4]: **OPTIONAL**
- It'd be very nice if you added a couple of images of the app under this nice description 😉

File: index.html [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+                        53.6l15.9-25.4h17l-16.6 26.3 17.3 26.9h-17.1l-16.5-26v26h-16V1.6l16-1.5"/></g></svg>
+                </div>
+            </section>
+            <section class="footer">
+                <img class="low-logo" src="./images/logo.png" alt="logo">
+                <div class="copyright lato">
+                    Emily Robertson, 2022. This is a dummy website for educational purposes only, based on a template by Cindy Shin.
+                </div> 
+            </section>
+        </footer>
```
Comment (MahmoudBakr23) [inline index.html:240]: - [ ] Please make sure to include the footer in the mobile screen version of your app
_I know that the design frame doesn't show it. However, it doesn't make sense to remove the footer of the page in the mobile screen version right?_ 😕

File: myscript.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+function displayRows() {
+  for (let i = 0; i < hiddenRows.length; i += 1) {
+    hiddenRows[i].classList.add('visible');
+    moreButton.classList.remove('more');
+  }
+}
+
+function displayMenu() {
+  mobileMenu.classList.toggle('mobile-menu');
+}
```
Comment (MahmoudBakr23) [inline myscript.js:112]: - [ ] Please double-check on the menu functionality when the user is on the about page. Currently, the user cannot roll back from the about page to the home page using the menu icon on the mobile screen version! 🙏

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @emyrue 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linters are correct
- Great use of JavaScript
- Nicely done following the design and setting up the CSS media queries
- Great presentation video

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Please see my comments down below_

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keroloussamy — APPROVED** [review]
Hi @emyrue, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**